web/styles/left_sidebar.css

```diff
@@ -974,6 +974,26 @@ li.top_left_scheduled_messages {
     }
 }
 
+.dm-markers-and-controls {
+    grid-area: markers-and-controls;
+    display: flex;
+    /* Present a uniform space between icons */
+    gap: 5px;
+    align-items: center;
+
+    .unread_mention_info {
+        /* Unset margin in favor of flex gap. */
+        margin: 0;
+    }
+
+    .unread_count {
+        /* Height is set here by the flexbox; this
+           decouples .unread_count from the app-wide
+           definition. */
+        height: 15px;
```

Comment:
Why is this 15px, rather than the value of the very similar rules for `.stream-markers-and-controls,` and friends? 

Also, is there a reason this is separate from that rule, or at least not next to it?